src/main/interface/cli.c

```diff
@@ -3075,6 +3085,24 @@ static void cliDefaults(char *cmdline)
     }
 }
 
+void cliPrintVarDefault(const clivalue_t *value)
+{
+    const pgRegistry_t *pg = pgFind(value->pgn);
+    if (pg) {
+        const char *defaultFormat = "Default value: ";
+        const int valueOffset = getValueOffset(value);
+        backupPgConfig(pg);
+        pgReset(pg);
```

Comment:
Sadly this does not work in some cases - target may use `targetConfiguration`  function that overrides pgReset values ... 
https://github.com/betaflight/betaflight/blob/dd151b3636b3d15f35c902863473298f9bb654fa/src/main/interface/cli.c#L3914

Reply:
Well that's frustrating! I was hoping to be more efficient and only backup, reset and restore the affected PG.

I'll rework it to wrap a single full backup/reset/restore around the possible multiple parameter matches.

Reply:
Turned out to not be that difficult - see additional commit.  Reworking to keep a copy of the defaults at all times is probably a worthwhile long-term approach.  But I'm not sure how much refactoring will be needed through the code.

Reply:
I think we probably best hold off on reworking this for now - a lot of this will change when universal targets are introduced, notably `targetConfiguration()` will be completely gone, so the original approach of this pull request will become viable then (at least for universal targets).